drivers/sensor/sbs_gauge/emul_sbs_gauge.c

```diff
@@ -21,7 +18,7 @@ LOG_MODULE_REGISTER(sbs_sbs_gauge);
 #include <zephyr/drivers/i2c_emul.h>
 #include <zephyr/sys/byteorder.h>
 
-#include <sbs_gauge.h>
+#include "../../fuel_gauge/sbs_gauge/sbs_gauge.h"
```

Comment:
I'm a bit confused by the include path and path to the .c file here, is this a fuel gauge or sensor emulated device? Both?

Reply:
Sorry, my IDE did this when I moved the file and I didn't catch it. Fixed